tdrs-backend/tdpservice/data_files/serializers.py

```diff
@@ -39,8 +40,28 @@ class Meta:
 
     def create(self, validated_data):
         """Create a new entry with a new version number."""
-        return DataFile.create_new_version(validated_data)
+        data_file = DataFile.create_new_version(validated_data)
+
+        # Determine the matching ClamAVFileScan for this DataFile.
+        av_scan = ClamAVFileScan.objects.filter(
```

Comment:
The `DataFile` doesn't get created until after the scan is complete and recorded so we have to retroactively update this if the creation was successful.